src/main/java/com/example/command/DeleteVertexCommand.java

```diff
@@ -47,7 +48,7 @@ public void undo() {
         graphController.update();
         graphController.setVertexPosition(vertex, x, y);
         graphController.colorVertex(vertex);
-        graphController.addVertexListeners(vertex);
+        graphController.onAddVertex(vertex);
```

Comment:
Is is intentional that after undo operation previous coloring is gone?

Reply:
It is not intentional, however it works the same on develop, so I think that this issue is not in the scope of this task